docker/ftest-docker-compose-v2/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '2'
 services:
   tomcat:
     env_file: envs
-    image: tutum/tomcat:7.0
```

Comment:
We have to add users explicitly by adding `ADD tomcat-users.xml conf/` in dockerfile, which is not provided in base image. ref - https://hub.docker.com/_/tomcat/